File: src/main/java/com/rody/student/Student.java
package com.rody.student;

public class Student {
    String id;
    String name;
    int math;
    int english;


    public Student(String name, int math, int english) {
        this.name   = name;
        this.math   = math;
        this.english= english;
    }

    // alt +insert
    public Student(String id, String name, int math, int english) {
        this.id = id;
        this.name = name;
        this.math = math;
        this.english = english;
    }

    public void print() {
        System.out.print(name+"\t"+math+"\t"+english+
                "\t"+ getAverage()+ "\t"+((getAverage() > 60) ? "PASS" :"Failed"));

        char grading='F';
        switch (getAverage()/10) {
            case 10:
            case 9:
                grading='A';
                break;
            case 8:
                grading='B';
                break;
            case 7:
                grading='C';
                break;
            case 6:
                grading='D';
                break;
            default:
                break;

        }
        System.out.println("\t"+ grading);
        /*
        if (getAverage()>=60){
            System.out.println("\tPASS");
        }
        else {
            System.out.println("\tFailed");
        }
         */
    }

    public int highest(){
        /*
        int max=0;
        if (english > math) {
            max=english;
        }
        else {
            max= math;
        }
        return max;
         */
        return (english > math) ? english : math ;
    }

    public int getAverage() {
        return (english+math)/2;
    }
}

File: src/main/kotlin/com/rody/student/Student.kt
package com.kotlin.rody.student

import java.util.*

fun main() {
    //userInput();
    val stu = Student("Rody", 50, 90)
    stu.print()
    println("High score: ${stu.highest()}")
}

private fun userInput() {
    val scanner = Scanner(System.`in`)
    print("Please sudent's name:")
    var name = scanner.next()

    print("Please sudent's enlish:")
    var english = scanner.nextInt();

    print("Please sudent's math:")
    var math = scanner.nextInt();

    val stu = Student(name, english, math)
    stu.print()
}

class Student(var name: String, var english:Int, var math:Int) {
    fun print() {
        print(name+"\t"+english+"\t"+math+"" +
                "\t"+ getAverage() +"\t"+ if (getAverage()>=60) "PASS" else "FAILED")
        println("\t"+grading())
    }

    fun grading()= when(getAverage()){
                    in 90..100 -> 'A'
                    in 80..89  -> 'B'
                    in 70..79  -> 'C'
                    in 60..69  -> 'D'
                    else -> 'F'
                }
    /*
    fun grading():Char {
        var grading: Char ='F'
        grading= when(getAverage()){
                    in 90..100 -> 'A'
                    in 80..89  -> 'B'
                    in 70..79  -> 'C'
                    in 60..69  -> 'D'
                    else -> 'F'
                 }
        return grading;
    }

     */

    fun highest()=if(english>math) {
                    println("English")
                    english
                } else {
                    println("Math")
                    math
                }
    /*
    fun highest():Int{
        var max = if(english>math) {
            println("English")
            english
        } else {
            println("Math")
            math
        }
        return max
    }

     */

    fun getAverage()=(english+math)/2
    /*
    fun getAverage():Int {
        return (english+math)/2

    }

     */

}